prep-html-links/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="http://localhost:5500/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```

Comment:
same as above, you don't want localhost, just the relative path from this file to the contact-us.html